_posts/2017-02-09-rapid-mysql-imports-docker.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: "Dramatically improve MySQL database import time on Docker stacks"
+date: 2017-02-09
+author: Kosta Harlan
+tags: drupal drupal-planet docker
+summary: "How to dramatically improve MySQL database import time on Dockerized Drupal stacks"
+featured_image: "/blog/sea-ocean-boats-port.jpg"
+featured_image_alt: "Docker, ships, the sea, etc"
+featured_image_height: "983px"
+featured_image_width: "1474px"
+drupal_planet_summary: |
+  A tutorial to show how you can use data volume restore for MySQL on Dockerized Drupal stacks to dramatically speed up import times, and notes on how to integrate this with your development and continuous integration practices.
+---
+
+If you're a Drupal developer, the process of (re)importing a seed database locally or for automated testing builds is something you do many times per day. If you're using Docker for your local development or continuous integration environments, here's a tip to dramatically speed up the import of databases, especially large ones.
+
+## The usual way
+
+The typical way to import a database is with using the `mysql` command to process a SQL dump file:
```

Comment:
Suggest changing to "the typical method of importing a database is to use the `mysql` command to..."